examples/multi-language/python/wordcount_external.py

```diff
@@ -0,0 +1,106 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+
+import apache_beam as beam
+from apache_beam.io import ReadFromText
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.transforms.external_transform_provider import ExternalTransformProvider
+from apache_beam.typehints.row_type import RowTypeConstraint
+"""A Python multi-language pipeline that counts words.
+
+This pipeline reads an input text file then extracts and counts the words using Java SDK SchemaTransforms provided in
+`ExtractWordsProvider`, `JavaCountProvider`, and `WriteWordsProvider`. Wrappers for these transforms are dynamically
+provided in Python via the `ExternalTransformProvider` API.
+
+Before running this program, make sure the expansion service is up and running. You can do so with the command:
+$ ./gradlew examples:multi-language:runExpansionService -PexpansionPort=<PORT>
```

Comment:
we should provide the command without using gradelw since the users who just install the beam do not use this.

Reply:
Thanks for the catch, this was an old comment. Removed it